examples/simple-proposals.yaml

```diff
@@ -38,8 +38,8 @@
     receiver: https://alice.example/
     resourceClassifiedAs: https://www.wikidata.org/wiki/Q4917 # US Dollar
     resourceQuantity:
-      qudt:unit: unit:Number
-      qudt:numericValue: 15
+      om2:hasUnit: om2:one
```

Comment:
s/b http://www.ontology-of-units-of-measure.org/resource/om-2/UnitedStatesDollar ?